services/cases-api/lambdas/createCase.js

```diff
@@ -87,6 +89,20 @@ export async function main(event) {
   });
 }
 
+/**
+ * @param {number} futureHours hours in the future
+ * @returns {number} a future date in seconds
+ */
+function generateExpirationTime(futureHours) {
```

Comment:
A more sutible naming here would be calculateCaseExpirationTimeInSeconds. Since generateExpirationTime does not give me a hint on what the scope of the expiration time is applied to.

Reply:
also the argument here is kind of vague, why not only call it hours?